JavaAssignments/src/saket/LeapYear.java

```diff
@@ -0,0 +1,43 @@
+package saket;
+
+import java.util.Scanner;
+
+public class LeapYear {
+
+	public static void main(String[] args) {
+
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter the Year");
+
+		int year = sc.nextInt();
+
+		boolean flag = leap(year);
+
+		display(flag);
+
+	}
+
+	static boolean leap(int yr) {
+		boolean flag = false;
+		if (yr % 4 == 0) {
+			flag = true;
+
+		}
+		if (yr % 100 == 0) {
```

Comment:
same code can be written as 

if(year % 100 == 0)
{
      if( year % 400 == 0)
     {
         flag = true;
     }else{
       flag = false;  
    }     
}